### PR TITLE
Add CLI section to admin docs

### PR DESCRIPTION
**Pull request by codyjlandstrom**

### CODE REVIEW

File: src/content/admin/dashboard.mdx [hunk 1]
```diff
@@ -150,7 +150,30 @@ This table provides a consolidated view about Preview Environments, including:
...
+  - **Any Version**: No restriction, developers can use any CLI version
+  - **Current Version (e.g., 3.2.x)**: Restricts developers to the latest CLI version packaged with the cluster
+  - **Previous Version (e.g., 3.1.x)**: Supports the latest version and the immediately prior minor version
+  - **Two Previous Versions (e.g., 3.0.x - 3.2.x)**: Supports the latest version and the previous two minor versions
+
+**CLI Version Enforcement**
+
+- Commands fail if a developer uses a CLI version outside the allowed range, displaying an error prompting them to update their CLI.
+
+**Feature Flag for Development**
```
Comment (pchico83) [inline src/content/admin/dashboard.mdx:174]: This is an internal feature flag, I wouldn't document it for now.

File: src/content/admin/dashboard.mdx [hunk 2]
```diff
@@ -150,7 +150,30 @@ This table provides a consolidated view about Preview Environments, including:
...
+
+- Set the minimum supported CLI version in the Okteto Admin Dashboard with these options:
+  - **Any Version**: No restriction, developers can use any CLI version
+  - **Current Version (e.g., 3.2.x)**: Restricts developers to the latest CLI version packaged with the cluster
+  - **Previous Version (e.g., 3.1.x)**: Supports the latest version and the immediately prior minor version
+  - **Two Previous Versions (e.g., 3.0.x - 3.2.x)**: Supports the latest version and the previous two minor versions
+
+**CLI Version Enforcement**
+
+- Commands fail if a developer uses a CLI version outside the allowed range, displaying an error prompting them to update their CLI.
```
Comment (pchico83) [inline src/content/admin/dashboard.mdx:172]: It doesn't fail for the range, newer versions are always supported

File: src/content/core/remote-execution.mdx [hunk 3]
```diff
@@ -12,7 +12,9 @@ By default, your commands run locally in your machine, but we recommend that you
 
 ## Enable remote execution
 
-Enable remote execution by setting the `remote` field to `true` in the deploy/destroy section of your Okteto Manifest:
+You can enable remote execution by default for all applications on your Okteto cluster by [following these instructions](#enable-remote-execution-by-default).
+
+To use remove execution per application, you can enable it by setting the `remote` field to `true` in the deploy/destroy section of your Okteto Manifest:
```
Comment (pchico83) [inline src/content/core/remote-execution.mdx:17]:
```suggestion
To use remote execution per application, you can enable it by setting the `remote` field to `true` in the deploy/destroy section of your Okteto Manifest:
```

Comment (pchico83) [reply src/content/core/remote-execution.mdx:17]: We already have this paragraph to document this:
https://deploy-preview-935--okteto-docs.netlify.app/docs/1.27/core/remote-execution/#enable-remote-execution-by-default
We should refactor this info to have a single place that explains how to enable remote execution by default

File: src/content/admin/dashboard.mdx (hunk 2 above)
Comment (ifbyol) [reply src/content/admin/dashboard.mdx:172]: Also, I think we should mention here that this starts to apply from `3.2` and above. I mean, `3.1` doesn't have the code validating the backend version, so even if the backend is forcing to use only `3.2`, users might still use `3.1` CLI

File: src/content/core/remote-execution.mdx (hunk 3 above)
Comment (codyjlandstrom) [reply src/content/core/remote-execution.mdx:17]: Bringing that section up to consolidate

**Review by pchico83 — APPROVED** [review]

**Review by viddo — APPROVED** [review]

**Pull request merged**